### PR TITLE
Refactored Func.__repr__().

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

**Comment by timgraham** [comment]
It would be nice to solve the issue of verbose `__repr__` methods in subclasses but as Simon said on the linked issue, using `extra` for this might not be a good solution since it mixes the concern of `__repr__` with variables that are meant to be interpolated into the template. Perhaps a new method for retrieving additional key/value pairs for `__repr__` would allow also simplifying things like `Count.__repr__`.

**Comment by felixxm** [comment]
@timgraham You're right, thanks for the advice 👍. I updated PR.

File: django/db/models/aggregates.py [hunk 1]
```diff
@@ -36,6 +36,20 @@ def default_alias(self):
     def get_group_by_cols(self):
         return []
 
+    def get_options(self):
```
Comment (timgraham) [inline django/db/models/aggregates.py]: I'd sort `__repr__` under `__init__` and name this something like `_get_repr_options` and add a docstring to the base class implementation like `Return a dict of extra __init__() options to include in the repr.`
Did you consider trying to put this higher up the inheritance chain than `Aggregate`?

**Comment by felixxm** [comment]
@timgraham Thanks for review. I moved the changes to `Func`.

**Pull request merged**